modules/generic-scala3/src/main/scala/pureconfig/generic/scala3/EnumDerivationUtils.scala

```diff
@@ -0,0 +1,33 @@
+package pureconfig
+package generic
+package scala3
+
+import scala.deriving.Mirror
+
+import pureconfig.generic.derivation._
+
+private[generic] object EnumDerivationUtils {
```

Comment:
nit: this doesn't look like utilities at all, this is more like a facade for all enum derivation. What do you think of simply naming it `EnumDerivation`?

```suggestion
private[generic] object EnumDerivation {
```